blockly/build.gradle

```diff
@@ -4,11 +4,12 @@ plugins {
 
 
 dependencies {
-    implementation project(":dungeon")
+    implementation project(':dungeon')
```

Comment:
ich kann verstehen, dass dich das massiv juckt. bitte aber in diesem pr nur dinge, die hier auch hingehören. das hier gehört jedenfalls nicht hierher.

Reply:
wir hatten mal gesagt, wenn "andere" sachen auffallen, die aber nicht unmittelbar mit dem jeweiligen pr zu tun haben, dürfen wir diese auch ändern.

aber ich mache das rückgängig, moment.

Reply:
echt? war ich da dabei? ich finde dieses "lass mal eben schnell mitmachen" absolut furchtbar.